Documentation/network/pod-mac-address.rst

```diff
@@ -0,0 +1,59 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _pod_mac_address:
+
+************************************
+Use a Specific MAC Address for a Pod
+************************************
+
+Some applications bind software licenses to network interface MAC addresses.
+Cilium provides the ability to specific MAC addresses for pods
+at deploy time instead of letting the operating system allocate them.
+
+
+Configuring the address
+#######################
+
+Cilium will configure the MAC address for the primary interface inside a
+Pod if you specify the MAC address in the ``cni.cilium.io/mac-address`` before
+deploying the Pod. This MAC address is isolated to the container so it will
```

Comment:
nit: 
```suggestion
Pod if you specify the MAC address in the ``cni.cilium.io/mac-address`` annotation before
deploying the Pod. This MAC address is isolated to the container so it will
```